src/Lecture2.hs

```diff
@@ -190,8 +300,14 @@ verify that.
 >>> merge [1, 2, 4] [3, 7]
 [1,2,3,4,7]
 -}
+
+--merge :: Ord a => [a] -> [a] -> [a]
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge a [] = a
+merge [] b = b
+merge f@(a:as) s@(b:bs)
+    | a < b = a : merge as s
+    | otherwise = b : merge f bs
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️